### DISCUSSION TITLE
NodeのDLをする時になにをしたか

### DISCUSSION THREAD

**Discussion by NoRio-kd** [TeamC-by-TNI/Felizo#66] [General]
## ノードのDL

下記のコマンドを順番に実行
1. `node install`  
2. `npm install`  
3. `npm run dev`

### 個別の Vite のインストールが必要な場合
npm install --save-dev vite laravel-vite-plugin

### module not foundが出た場合
rm -rf node_modules package-lock.json

**Comment by mikiIDE** [comment]
NodeをUbuntuでインストールしました！
http://localhost:5173/でLaravelとViteのサイト？みたいなやつは表示されるんですが
やっぱりlovalhostは接続が拒否されます…
でもこの問題は二日前から堂々巡りしてているので一旦区切ってフロントの調整に全力出します！
コマンド共有してもらったのにすいません！
